### DISCUSSION TITLE
Prefix view: Child IPs tab is hidden for Container prefixes

### DISCUSSION THREAD

**Discussion by sdktr** [netbox-community/netbox#7088] [Q&A] [answered]
In https://github.com/netbox-community/netbox/blob/fd16c47d2e1d9b3b4a0a9e39de4f3dea759fb250/netbox/templates/ipam/prefix/base.html#L28 it is determined that for Prefixes of type Container, no Child IP tab should be shown on the prefix page.
![image](https://user-images.githubusercontent.com/19833687/131485659-df862ca9-742d-45bd-878b-95d687b8e545.png)
![image](https://user-images.githubusercontent.com/19833687/131485547-6e222a98-0feb-4c6e-a181-3306fbc5a147.png)

I'd like to propose we eighter skip this check, or relax it so that a Pool will show this tab regardless of status. The use case is that we model a pool for VPN clients as a prefix with type Container and 'is_pool: true'. 

What are other peoples opinion on this?

**Comment by candlerb** [comment, marked as answer]
Already open ticket: #5925

Workaround: change the prefix status from "container" to "reserved".

"is_pool" only affects whether the first and last IP address are excluded when calculating the percentage utilisation.

**Reply by sdktr** [reply]
There propably is a reason this 'container' condition was added. The rationale may be that a Container should not contain IP addresses directly. I disagree with that and would propose we show Child IPs for containers as well.

**Reply by candlerb** [reply]
I agree with you, because there is no logic in NetBox which prevents you creating an IP Address directly with a Container Prefix: and if you do, it appears to vanish in the GUI. That is, you can't navigate to it, and it doesn't count towards prefix utilisation.

You can always +1 on [the ticket](https://github.com/netbox-community/netbox/issues/5925#issuecomment-860041920).
